lib/svg.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const svgoLoader = {
+    loader: 'svgo-loader',
+    options: {
+        plugins: [
+            { removeTitle: true },
+            { removeDimensions: false },
+            { removeViewBox: false },
+            { cleanupIDs: false },
+        ],
+    },
+};
+
+module.exports = (pluginOptions = {}) => (nextConfig = {}) => ({
+    ...nextConfig,
+    ...{
+        webpack(config, options) {
+            const { dev, isServer } = options;
+            const { assetPrefix = '' } = nextConfig;
+            const { inline = false, ...userOptions } = pluginOptions;
+
+            const inlineLoaders = {
```

Comment:
I would prefer to actually have an if/else here:

```js
if (inline) {
    config.module.rules.push(...);
} else {
     config.module.rules.push(...);
}
```

and get rid of the intermediate `const` that might or not be used.